packages/client/src/views/CorrectionForm/CorrectionSummary.tsx

```diff
@@ -172,6 +149,21 @@ class CorrectionSummaryComponent extends React.Component<IFullProps, IState> {
       declaration: { event },
       userRole
     } = this.props
+
+    const currencySymbol = getCurrencySymbol(
+      this.props.offlineResources.config.CURRENCY
+    )
```

Comment:
The currency symbol is probably a string right? Then let's type the `getCurrencySymbol` properly by setting it's return type.